charts/node/templates/service.yaml

```diff
@@ -150,6 +150,11 @@ spec:
     - name: p2p
       port: {{ $.Values.node.perNodeServices.paraP2pService.port | int }}
       targetPort: 30334
+    {{- if $.Values.node.perNodeServices.paraP2pService.ws.enabled }}
+    - name: ws
+      port: {{ $.Values.node.perNodeServices.paraP2pService.ws.port | int }}
+      targetPort: 30334
```

Comment:
Same targetPort as the p2p one, is that right?

Reply:
Yes. the p2p port is 30333 whereas p2p-ws is 30334, both for relaychain and parachain services.

Reply:
sorry I checked again, you're right. The port should be changed.
@PierreBesson the parachain p2p service port is set to 30334, while this port for relaychain is 30333. could we change the port in parachain to 30333 as well as those are two separate services and could have the same port set?
otherwise I would keep 30334 for parachain-p2p and 30335 for parachain-p2p-ws.